webpack.config.js

```diff
@@ -55,4 +57,13 @@ module.exports = {
         new webpack.optimize.UglifyJsPlugin(),
         new webpack.optimize.DedupePlugin(),
     ],
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components|javascripts-legacy|components|vendor)/,
```

Comment:
Probably not that simple but is this the same as `include: 'static/src/javascripts'` (or whatever the webpack incantation is)? I wonder if inclusion is safer than exclusion anyhow?